sky/backends/backend_utils.py

```diff
@@ -1715,7 +1764,9 @@ def _update_cluster_status_no_lock(
 def _update_cluster_status(
         cluster_name: str,
         acquire_per_cluster_status_lock: bool) -> Optional[Dict[str, Any]]:
-    """Update the cluster status by checking ray cluster and real status from cloud.
+    """Update the cluster identity and status.
```

Comment:
```suggestion
    """Update the cluster status and owner identity.
```

Reply:
I changed the docstr a bit to `Check/update owner identity and update the cluster status.`. Wdyt?